Client/U2FExtensions.swift

```diff
@@ -238,6 +239,12 @@ class U2FExtensions: NSObject {
         currentMessageType = U2FMessageType.FIDO2Create
         currentHandle = handle
         
+        guard let tabId = tab?.id else {
```

Comment:
This is copy/pasted in like 4 places, and I see that these function implementations are pretty much identical, can we abstract these?

Also, this file is quite massive and contains almost no comments, moving forward it may be good to start addign some to newly added code.

Reply:
on it 👍